xml/art_solution_automation.xml

```diff
@@ -0,0 +1,438 @@
+    <para>
+      The &salt; formula can be used in these scenarios:
+    </para>
+    <itemizedlist>
+      <title>Scenarios for <package>sapnwbootstrap-formula</package></title>
+      <listitem>
+        <para>Enqueue replication server version 1</para>
+      </listitem>
+      <listitem>
+        <para>Enqueue replication server version 2</para>
+      </listitem>
+      <listitem>
+        <para>Azure, AWS, and GCP automated adjustments for each provider best-practices</para>
+      </listitem>
+    </itemizedlist>
+    <example>
+      <title>Installation</title>
+      <screen>zypper addrepo --refresh obs://network:ha-clustering:Factory ha-clustering
+zypper install --repo ha-clustering sapnwbootstrap-formula</screen>
+    </example>
+
+    <itemizedlist>
+      <title>Best Practises</title>
+      <listitem>
+        <para><link xlink:href="https://documentation.suse.com/sbp/all/html/SAP_HA740_SetupGuide_AWS/index.html"
+          >&netweaver; &ha; Cluster 7.40 for the AWS Cloud - Setup Guide (Code 12)</link></para>
+      </listitem>
+      <listitem>
+        <para><link xlink:href="https://www.suse.com/media/white-paper/sap_netweaver_availability_cluster_740_setup_guide.pdf"
+          >&netweaver; &ha; Cluster 7.40 - Setup Guide (Code 12)</link></para>
+      </listitem>
+      <listitem>
+        <para><link xlink:href="https://www.suse.com/media/white-paper/sap_netweaver_availability_cluster_740_setup_guide.pdf"
+          >Simple Stack - &netweaver; &ha; on &sle; 12 (SP1 or newer)</link></para>
+      </listitem>
+      <listitem>
+        <para><link xlink:href="https://documentation.suse.com/sbp/all/html/SAP_S4HA10_SetupGuide-SLE12/index.html"
+          >&s4h; - Enqueue Replication 2 &ha; Cluster - Setup Guide</link></para>
+      </listitem>
+      <listitem>
+        <para><link xlink:href="https://docs.microsoft.com/en-us/azure/virtual-machines/workloads/sap/high-availability-guide-suse"
+          >Azure</link></para>
+      </listitem>
+    </itemizedlist>
+  </sect1>
+
+  <sect1 xml:id="sec-sol-autom-ha-cluster-config">
+    <title>&ha; Cluster Automated Configuration</title>
+    <para>
+      &salt; formula <package>habootstrap-formula</package> for bootstrapping and managing
+      a high availability cluster using &crmshell;.
+    </para>
+    <formalpara>
+      <title>Link:</title>
+      <para>See <link xlink:href="https://github.com/SUSE/habootstrap-formula"/>.</para>
+    </formalpara>
+    <itemizedlist>
+      <title>Properties for <package>habootstrap-formula</package></title>
+      <listitem>
+        <para>
+          Rund on premises and on Cloud service providers (Azure, GCP, AWS).
+        </para>
+      </listitem>
+      <listitem>
+        <para>
+          Only passwordless GPG keys are supported. Already existing keys cannot be used.
+        </para>
+      </listitem>
+      <listitem>
+        <para>
+          If a masterless approach is used (as in the current automated deployment) the GPG
```

Comment:
Again, if we are talking about `ssh` keys, being in a master/minion or masterless approach doesn't really matter.

Reply:
Ok, I see. I rephrased this sentence and removed this "masterless approach" thing:

    Import all private GPG keys in all the nodes (as in the current automated
    deployment). This might require to copy and paste the keys.

Reply:
Ok, removed as discussed.